components/movementsensor/gpsrtk/gpsrtk.go

```diff
@@ -65,10 +64,14 @@ type SerialConfig struct {
 	SerialBaudRate           int    `json:"serial_baud_rate,omitempty"`
 	SerialCorrectionPath     string `json:"serial_correction_path,omitempty"`
 	SerialCorrectionBaudRate int    `json:"serial_correction_baud_rate,omitempty"`
+
+	// TestChan is a fake "serial" path for test use only
+	TestChan chan []uint8 `json:"-"`
```

Comment:
if it's only for testing, should we move this to test? Can't we use a fake path for testing? Maybe I'm not understanding the reason for this Test channel.

Reply:
This doesn't render on the app, by the way, it does not create a json config field there.